persistence/src/main/java/tennisclub/entity/Booking.java

```diff
@@ -14,6 +16,11 @@
 public class Booking extends Event {
 
     @ManyToMany
+    @JoinTable(
+        name = "userBooking",
```

Comment:
Shouldn't we actually use underscore_case instead of camelCase? It's just a matter of opinion so it's not really that important 😊

Reply:
My thoughts on this were that we are using Java with camelCase and if we accidentally persist an attribute consisted of multiple words that we forget to name, it will be named using camelCase. Therefore to ensure consistency throughout the database, I decided not to use snake_case. 

It should be just a matter of preference, really (even though snake_case is more common in the databases, at least from my experience), however, if we decide to go for snake_case, I'm happy to do a rewrite and check everything :)

Reply:
I hope I understood you correctly; correct me if I'm wrong _(I apologise beforehand if I didn't understand you)_ - so when we try to create a new attribute that consists of multiple words _(written in camelCase in Java)_, it would be named using 🐫Case in the database? 🤔

If that's the case though, I don't think we need to be concerned about it, as when I tried to run the `main` method, it converts all of the 🐫Case attributes to snake_case actually (for example, try to take a look at attribute `startTime` in the `Event` Entity. So by default it is actually 🐍_case, and if we forget to name an attribute, it is okay as it will be by default 🐍_case _(and would actually create an inconsistency if we are going to use 🐫Case)_.

Correct me, if I am wrong though, I am not well versed in this area and as you said, I have just encountered with 🐍_case in databases actually 😁.

Reply:
Blimey, you are right, many apologies! I wasn't aware of Spring actually converting camelCase attributes to snake_case and I was assuming the behaviour (why the hell would it do such a thing?? 😄  )... IMHO it has to create more confusion than help as the conversion cannot be always perfect (e.g. `publicAPI` converts to `publicapi`), but I guess that's just one of the many Spring curiosities 😄 

What's more confusing is that even if I use `@JointTable(name = "userBooking")` and run the application, it actually gets converted to `user_booking`, so technically the code is consistent, but I'm definitely going to rewrite the names use snake_case, just to be sure.